content/docs/recipes/upgrade-kafka.md

```diff
@@ -0,0 +1,289 @@
+.
+└── kafka (Operator-Version: "kafka-1.2.0" Active-Plan: "deploy")
+    ├── Plan deploy (serial strategy) [IN_PROGRESS]
+    │   └── Phase deploy-kafka (serial strategy) [IN_PROGRESS]
+    │       └── Step deploy [IN_PROGRESS]
+    ├── Plan mirrormaker (serial strategy) [NOT ACTIVE]
+    │   └── Phase deploy-mirror-maker (serial strategy) [NOT ACTIVE]
+    │       └── Step deploy [NOT ACTIVE]
+    └── Plan not-allowed (serial strategy) [NOT ACTIVE]
+        └── Phase not-allowed (serial strategy) [NOT ACTIVE]
+            └── Step not-allowed [NOT ACTIVE]
+```
+
+Once the pods are ready, which can be checked using next command
+```
+kubectl get pods -n kudo-kafka
+```
+the expected output is the list of the pods
+
+```
+NAME                    READY   STATUS    RESTARTS   AGE
+kafka-kafka-0           2/2     Running   2          7m9s
+kafka-kafka-1           2/2     Running   0          5m59s
+kafka-kafka-2           2/2     Running   0          5m21s
+```
+
+Once the are passing the both readiness and liveness checks the plan should be complete
+```
+kubectl kudo plan status --instance=kafka -n kudo-kafka
+```
+
+expected output should show the `deploy` plan complete and the `Operator-Version` to be `kafka-1.2.0`
+
+```
+Plan(s) for "kafka" in namespace "kudo-kafka":
+.
+└── kafka (Operator-Version: "kafka-1.2.0" Active-Plan: "deploy")
+    ├── Plan deploy (serial strategy) [COMPLETE]
+    │   └── Phase deploy-kafka (serial strategy) [COMPLETE]
+    │       └── Step deploy [COMPLETE]
+    ├── Plan mirrormaker (serial strategy) [NOT ACTIVE]
+    │   └── Phase deploy-mirror-maker (serial strategy) [NOT ACTIVE]
+    │       └── Step deploy [NOT ACTIVE]
+    └── Plan not-allowed (serial strategy) [NOT ACTIVE]
+        └── Phase not-allowed (serial strategy) [NOT ACTIVE]
+            └── Step not-allowed [NOT ACTIVE]
+```
+
+#### 10. Verify the Kafka version through the pod logs
+```
+kubectl logs kafka-kafka-0 -c k8skafka -n kudo-kafka | grep "Kafka version:"
+```
+
+The expected output is the Kafka version and that should be the upgraded version `2.4.0`
+
+```
+[2020-01-14 15:13:20,664] INFO Kafka version: 2.4.0 (org.apache.kafka.common.utils.AppInfoParser)
+```
+
+#### 11. Verify the Kafka version through the container version
+
+```
+kubectl get pods kafka-kafka-0 -n kudo-kafka -o json | jq -r '.spec.containers[].image'
+```
+
+the expected output will show the container images used by the KUDO Kafka pods. The `mesosphere/kafka` image should be the upgraded version `1.1.0-2.4.0`
+
+```
+quay.io/prometheus/node-exporter:v0.18.1
+mesosphere/kafka:1.1.0-2.4.0
+```
+
+#### 12. Verify the Kafka version through the app version of the installed KUDO Instance
+
+```
+kubectl get instances.kudo.dev kafka -n kudo-kafka -o json | jq -r .spec.operatorVersion.name | xargs kubectl get operatorversion -n kudo-kafka -o json | jq -r .spec.appVersion
+```
+
+the expected output should be the app version specified in the operator version of the instance
```

Comment:
capitalize

Reply:
SGTM! addressed the other PR comments also